spec/svg_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe 'SVG files' do
+  Dir[Rails.root.join('**/*.svg')].each do |svg_path|
+    relative_path = svg_path.sub(Rails.root.to_s, '')
+
+    describe relative_path do
+      it 'does not contain inline style tags' do
```

Comment:
need test! for future context, might be helpful to include something in this spec description or in a comment in this spec about the reason for not having inline styles.